src/MatrixTypes_Trilinos.f90

```diff
@@ -397,7 +400,19 @@ SUBROUTINE matvec_TrilinosVector(thisMatrix,trans,alpha,x,beta,y,uplo,diag,incx_
             modName//'::'//myName//' - This interface is not available.')
       ENDSELECT
     ENDSUBROUTINE matvec_TrilinosVector
-    
+!
+!-------------------------------------------------------------------------------
+!> @brief zero all the elements of the matrix
+!> @param matrix declare the matrix type to act on
+!>
+!>
+    SUBROUTINE zeroentries_TrilinosMatrixType(matrix)
+      CLASS(TrilinosMatrixType),INTENT(INOUT) :: matrix
+      CHARACTER(LEN=*),PARAMETER :: myName='matvec_MatrixTypeVectorType'
+      ! No need
+      CALL eMatrixType%raiseFatalError(modName//'::'//myName// &
```

Comment:
So if Trilinos already zeros the matrix... perhaps we just leave a REQUIRE statement in here so the "contract" for this routine is consistent across all implementations?